doc/manage_iot_hub.md

```diff
@@ -4,105 +4,28 @@ Before a device can communicate with IoT Hub, you must add details of that devic
 
 To add devices to your IoT hub and manage those devices, you can use either of:
```

Comment:
@abhipsaMisra I remember we've agreed to remove this.
We have better MSDN documentation for IoT Hub and DPS management.

Reply:
The text on the readme redirects to the MSDN document on Iot Hub and DPS